### PR TITLE
Stickers.py

### PR DESCRIPTION
**Pull request by bughunter0**
Stickers.py 

Return sticker id and Download and upload sticker files

### CODE REVIEW

File: pyrobot/plugins/stickers.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                   await tx.delete()   
+                   os.remove(file_path)
+                #   os.remove(zip_path) Removing zip file path
+             except Exception as error:
+                   print(error)
+ 
+          elif message.reply_to_message.sticker.is_animated is False:        
+             try : 
+                   tx = await message.reply_text("Downloading...")
+                   file_path = f"./DOWNLOADS/{message.chat.id}/png-{random_id}.png"
```
Comment (SpEcHiDe) [inline pyrobot/plugins/stickers.py:43]: same here, just give a directory path 😅

**Review by SpEcHiDe — CHANGES_REQUESTED** [review]

File: pyrobot/plugins/stickers.py (hunk 1 above)
Comment (bughunter0) [reply pyrobot/plugins/stickers.py:43]: Thanks , Trying to improve...

**Comment by bughunter0** [comment]
Is this the way ?

**Comment by bughunter0** [comment]
Updated😀

**Pull request closed without merging**